test/controllers/WebTest.java

```diff
@@ -163,4 +167,14 @@ private void checkSparkJobExceptionDetailElementForFlow2(JsonElement jobExceptio
 
     }
   }
-}
\ No newline at end of file
+
+  private void checkTonyJobExceptionDetailElementForFlow2(JsonElement jobExceptionDetail) {
+    JsonObject jobExceptionJsonObject = jobExceptionDetail.getAsJsonObject();
+    Assert.assertEquals(jobExceptionJsonObject.get(JsonKeys.NAME).getAsString(), "job_name_2_3");
+    Assert.assertEquals(jobExceptionJsonObject.get(JsonKeys.ID).getAsString(), "job_name_2_3");
+    Assert.assertEquals(jobExceptionJsonObject.get(JsonKeys.TYPE).getAsString(), "TONY");
+    JsonArray application = jobExceptionJsonObject.getAsJsonArray(JsonKeys.APPLICATIONS);
+    JsonObject applicationJsonObject = application.get(0).getAsJsonObject();
+    Assert.assertEquals(applicationJsonObject.get(JsonKeys.EXCEPTION_CLASSIFICATION).getAsString(), "TONY_INFRA_ERROR");
+    }
```

Comment:
Usage of `assertEquals` is incorrect. 
Kindly follow: `assertEquals(String message, Object expected, Object actual)`